apps/train_loc/lib/train_loc/manager/event.ex

```diff
@@ -6,10 +6,16 @@ defmodule TrainLoc.Manager.Event do
 
   alias TrainLoc.Manager.EventJsonParser
 
+  @type t() :: %__MODULE__{
```

Comment:
suggestion: what about adding the `path` to this struct, and using that in `TrainLoc.Manager` when deciding how to route the message?

Reply:
Yeah, I did that originally but changed it back since I wanted to avoid touching the old path. I think it's probably cleaner though.